quantum/secure.c

```diff
@@ -29,18 +30,29 @@ secure_status_t secure_get_status(void) {
 
 void secure_lock(void) {
     secure_status = SECURE_LOCKED;
+    secure_hook_quantum(secure_status);
 }
 
 void secure_unlock(void) {
     secure_status = SECURE_UNLOCKED;
     idle_time     = timer_read32();
+    secure_hook_quantum(secure_status);
 }
 
+void clear_keyboard(void);
```

Comment:
The issue here isnt that the header is included, its that `secure_request_unlock` calls the 2 functions. Thats the part that actually couples the implementation.